kr8s/_auth.py

```diff
@@ -163,6 +163,14 @@ async def _load_kubeconfig(self) -> None:
                     base64.b64decode(self._cluster["certificate-authority-data"])
                 )
                 self.server_ca_file = str(ca_file)
+        if "certificate-authority" in self._cluster:
+            if os.path.isabs(self._cluster["certificate-authority"]):
```

Comment:
Also another quick question, is this guaranteed to be an absolute path? Or would `os.path.isfile` be more suitable to handle symlinks?

Reply:
In our dex-issued kube config, the `certificate-authority` path is relative to the kube config. I think that might be standard. A relative path won't be readable by kr8s unless we prepend with the parent directory of the kube config, which is what we do in the `else` block. I've never actually seen `certificate-authority` with an absolute path. The only reason we added the `isabs` conditional was to cover that potential edge case. We didn't want to assume that every `certificate-authority` value was relative. But you're right -- it doesn't cover symlinks. I just pushed up a commit that switches from `isabs` to `isfile`.